bin/find-fixable-orphans.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env -S python3 -u
+#
+# This script needs to be run in a pyenv virtualenv with the Django project installed.
+#
+# Finds orphaned S3 objects that can be matched back to NZSL entries that are missing S3 objects.
+# Essentially finds one form of import error.
+#
+# Bang line above passes '-u' to python, for unbuffered output
+# Permissions required:
+#  psql - access to heroku app's postgres
+#  aws s3 - NZSL IAM access
+#  s3:GetObjectAcl permissions or READ_ACP access to the object
+#  https://docs.aws.amazon.com/cli/latest/reference/s3api/get-object-acl.html
+# For some commands you need to run this in a venv that has all the right Python site-packages.
+# TODO Convert this script to a Django Management Command
+
+import os
+import sys
+import subprocess
+import argparse
+from uuid import uuid4
+import boto3
+
+# Magic required to allow this script to use Signbank Django classes
+# This goes away if this script becomes a Django Management Command
+print("Importing site-packages environment", file=sys.stderr)
+print(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), file=sys.stderr)
+sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
```

Comment:
note: this feels a bit ugly, but my quick research doesn't suggest there's a cleaner way

the main alternative I could come up with was to use `pathlib` but I think that's only marginally better:

```
sys.path.append(str(Path(__file__).resolve().parents[2]))
```

Reply:
Yep, use of pathlib would make it cleaner. Changing it to a management command would remove the need entirely.

Reply:
pathlib definitely cleaner.  But I think I'll just change it to a management command. Means I can remove the caveat comment at the start about venv too.